frappe/installer.py

```diff
@@ -364,8 +364,9 @@ def check_if_ready_for_barracuda():
 			       "").format(x=site, sep2="\n"*2, sep="\n")
 
 			print_db_config(msg, expected_config_for_barracuda)
-			sys.exit(1)
-			# raise Exception, "MariaDB needs to be configured!"
+			raise frappe.exceptions.ImproperDBConfigurationError(
```

Comment:
the problem with leaving traces with users, is that most users get confused when left with a traceback. Lets not leave a trace with the user and show a clear message and exit with 1